src/components/simple-list.tsx

```diff
@@ -17,12 +26,21 @@ export const SimpleList: FC<SimpleListProps> = ({ listItems }) => {
       {listItems.map((item, i) => (
         <li
           key={i}
-          onClick={item.onClick}
-          className="flex justify-between border-b border-gray-100 px-4 py-3 last:border-none dark:border-neutral-800"
+          className={clsx('border-b border-gray-100 last:border-none', {
```

Comment:
The `dark:border-neutral-800` class need to be moved from the `<a>` to here in the `<li> `to work.